source/editor.c

```diff
@@ -9,13 +9,58 @@ which these 4 will have to undergo in order to prove themselves worthy? Stay tun
 #include <stdio.h>
 #include <string.h>
 #include <stdlib.h>
+#define MAX 10
 
 struct node{
 	char data;
 	struct node *next;
 	struct node *prev;
 };
 
+struct URNODE{
```

Comment:
What does URNODE mean? Ideally you'd want to use more descriptive, and not necessarily long names.